.github/workflows/test-lint.yml

```diff
@@ -16,6 +16,16 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
+        with:
+          android: false
+          dotnet: true
+          haskell: false
+          large-packages: false
+          docker-images: false
+          swap-storage: false
+
       - name: Setup Go, tools and caching
```

Comment:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/9c683923-743d-40bf-b7c2-9755fcab2e1c)

Reply:
(maybe it needs some better fix, It looks like, the linter cache is growing and growing)

Reply:
@michaljurecko  We are encoutering this issue in other PRs. Should we disable caching if the linter cache is growing? Currently I do not have much space to investigate it.

Reply:
@jachym-tousek-keboola Petr needs the fix, I am mergin it, you comment will be addresed in next PRs, if any.